src/bundle/Resources/views/themes/admin/account/change_password/form_fields.html.twig

```diff
@@ -1,17 +1,7 @@
 {# @todo move to view/user/preferences in the future #}
 
-{% use 'bootstrap_4_layout.html.twig' %}
-
 {% trans_default_domain 'change_user_password' %}
 
-{%- block form_errors -%}
-    {%- if errors|length > 0 -%}
-        {%- for error in errors -%}
-            <em class="ez-field__error">{{ error.message }}</em>
-        {%- endfor -%}
-    {%- endif -%}
-{%- endblock -%}
```

Comment:
Review note: Redundant, defined in `src/bundle/Resources/views/themes/admin/account/form_fields.html.twig`